src/macros.jl

```diff
@@ -503,6 +503,28 @@ and
 @transform(df, a = :x, b = :y)
 ```
 
+`@transform` uses the syntax `@byrow` to wrap transformations in
+the `ByRow` function wrapper from DataFrames, enabling broadcasting
+and more. For example, the call
+
+```
+@transform(df, y = @byrow :x == 1 ? "true" : "false)
+```
+
+becomes
+
+```
+transform(df, :x => ByRow(x -> x == 1 ? "true", "false") => :y)
```

Comment:
I think the point is that only `:x` will be broadcasted. Right? So if instead of e.g. `"true"` we would have `[1,2,3]` it would not be broadcasted.

Reply:
Yes. This also avoids the need for `Ref` with broadcasting. I will add a section in the docs (not the docstring) discussing this along with a few other gotchas.